src/commands/Tools/poll.ts

```diff
@@ -2,20 +2,59 @@ import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand'
 import { ApplyOptions } from '@skyra/decorators';
 import { KlasaMessage } from 'klasa';
 
+const NUMBER_OPTS = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣', '🔟'];
+
+const ALPHABET_OPTS = [
```

Comment:
This array is so obnoxiously long I'd like prettier to be ignored like in [char](https://github.com/skyra-project/char/blob/main/src/lib/CharUnicodeInfo/CategoriesValues.ts).

Reply:
Also, there are 25 elements, but the maximum is 20.